doc/cifarSC2011/advanced_theano.txt

```diff
@@ -179,96 +180,200 @@ Inplace optimization
 Profiling
 ---------
 
-- To replace the default mode with this mode, use the Theano flags ``mode=ProfileMode``
+- To replace the default mode with this mode, use the Theano flags ``profile=True``
 
-- To enable the memory profiling use the flags ``ProfileMode.profile_memory=True``
+- To enable the memory profiling use the flags ``profile=True,profile_memory=True``
 
-Theano output:
```

Comment:
There is nothing in the file that seems to mark it as such. Meaning that someone could find it and try to replicate what's in there. Is it that important to keep invalid documentation around to confuse users? The presentation was given in 2011. That's roughly 5 years ago!